packages/web/pages/players.tsx

```diff
@@ -77,24 +82,47 @@ const Players: React.FC<Props> = () => {
           resetFilter={resetFilter}
           totalCount={totalCount}
         />
-        {error && <Text>{`Error: ${error.message}`}</Text>}
-        {!error && players.length && (fetchingMore || !fetching) && (
+        {error ? <Text>{`Error: ${error.message}`}</Text> : null}
```

Comment:
¿Why do you prefer `x ? y : null` over `x && y`?

Reply:
This i feel is much clearer to read.